lib/modules/accountSwitcher.js

```diff
@@ -248,7 +248,7 @@ async function populateUser(container) {
 			if (userInfo.gold_expiration) {
 				const today = new Date();
 				const expDate = new Date(userInfo.gold_expiration * 1000); // s -> ms
-				$gilded.attr('title', `Until ${formatDate(expDate)} (${formatDateDiff(today, expDate)})`);
+				$gilded.attr('title', `${i18n("accountSwitcherGoldUntil")} ${formatDate(expDate)} (${formatDateDiff(today, expDate)})`);
```

Comment:
Nit: interpolation here too; in this case translators will have no idea what `"Until "` means. `"Until $1 ($2)"` is a bit better.

You can also add [descriptions](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/26703c80c01e27cd888e0bc180514ba3e5511172/locales/locales/en.json#L2835), which I would do here because it's fairly ambiguous (though they're certainly not required for most translations)